README.md

```diff
@@ -184,14 +186,31 @@ Some kinds of dependency fallbacks may be handled by [import maps](https://githu
 
 WebAssembly Modules are "compiled" and "instantiated" in a logically asynchronous way, based on their imports: Some WebAssembly implementations do nontrivial work at either phase, which is important to be able to shunt off into another thread. To integrate with the JavaScript module system, they will need to do the equivalent of a top-level await. See the [WebAssembly ESM integration proposal](https://github.com/webassembly/esm-integration) for more details.
 
+## New syntax
+
+This proposal allows `await` to be used at the top-level of a module, outside of functions.
+
+This proposal adds a new type of `import` statement, which is `import await`. `import await` can be used both modules which do and don't contain a top-level `await`. However, `import` statements may not be used on modules with top-level `await`--that misuse causes an error during the Linking phase. Some forms of `import await`:
```

Comment:
> `import` statement

Is [`ImportDeclaration`](https://tc39.github.io/ecma262/#prod-ImportDeclaration) a [`Statement`](https://tc39.github.io/ecma262/#prod-Statement)?

Reply:
Eh, it's an explainer.